src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -1130,6 +1130,55 @@ public async Task RemoveSectionCompleted(Guid applicationId, Guid applicationSec
             }
         }
 
+        public async Task<List<ApplicationOversightDetails>> GetOversightsPending()
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                return (await connection.QueryAsync<ApplicationOversightDetails>(@"SELECT 
+                            apply.Id AS Id,
+                            apply.ApplicationId AS ApplicationId,
+							 org.Name AS OrganisationName,
+					        JSON_VALUE(apply.ApplyData, '$.ApplyDetails.UKPRN') AS Ukprn,
+                            REPLACE(JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ProviderRouteName'),' provider','') AS ProviderRoute,
+							JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ReferenceNumber') AS ApplicationReferenceNumber,
+                            JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ApplicationSubmittedOn') AS ApplicationSubmittedDate,
+							apply.OversightStatus,
+							Apply.ApplicationDeterminedDate
+                              FROM Apply apply
+	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
+	                      WHERE apply.DeletedAt IS NULL
+                          and GatewayReviewStatus  in ('Pass','Fail')
```

Comment:
This would have to be Pass for the other statuses to be activated

Reply:
Yes, it was a pass originally, but the state diagram shows it flows to oversight even when it's a fail.... I'll change it to pass anyway, we all know it actually should never appear if gateway is a fail